quinn-proto/src/connection/assembler.rs

```diff
@@ -29,6 +29,16 @@ impl Assembler {
         Self::default()
     }
 
+    /// Reset to the initial state
+    pub(super) fn reinit(&mut self) {
```

Comment:
I'm not super happy with `reinit`. `reset` is already used for other purposes. I had `recycle` at some point but I like that even less

Reply:
Maybe `clear`? That seems logically equivalent. I don't mind `reinit`, though.

Reply:
Assembler already has a clear(), and it doesn't clear all the fields

Reply:
Then `reinit` seems fine.